frontend/src/UserList.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react';
+import {IPlayer} from './types';
+import styled from 'styled-components';
+import EmojiButton from './EmojiButton';
+import {WebsocketContext} from './WebsocketProvider';
+
+const Container = styled.div`
+  width: 80%;
+  border: 1px solid #eee;
+  border-radius: 4px;
+  padding: 1em;
+`;
+
+const Row = styled.div`
+  display: table-row;
+  border-collapse: separate;
+`;
+
+const Cell = styled.div`
+  display: table-cell;
+  border-style: solid;
+  border-color: transparent;
+  border-width: 0.2em 1em;
+`;
+
+const FullWidthCell = styled.div`
+  text-align: center;
+`;
+
+const Heading = styled(FullWidthCell)`
+  font-size: 1.2em;
+  padding: 0.5em;
+  font-weight: 200;
+`;
+
+const Empty = styled(FullWidthCell)`
+  font-style: italic;
+  color: #555;
+`;
+
+const ButtonContainer = styled.div`
+  display: flex;
+  flex-direciton: row;
+  justify-content: flex-end;
+  width: 10em;
+  & > * {
+    margin: 0px 0.3em;
+  }
+`;
+
+type RowProps = {
+  player: IPlayer;
+  actions: JSX.Element;
+};
+const UserRow = (props: RowProps) => (
+  <Row key={props.player.name}>
+    <Cell style={{width: '50%'}}>{props.player.name}</Cell>
+    <Cell style={{width: '50%'}}>Rank {props.player.level}</Cell>
+    <Cell>{props.actions}</Cell>
+  </Row>
+);
+
+type Props = {
+  currentPlayer: IPlayer;
```

Comment:
Can we still include the `landlord` information in this component? I don't think players will know to look in the dropdown in continuation settings

Reply:
Updated in fdcce4883c2a56bff0a886edc7aa3ccbbcf251fe

<img width="599" alt="Screen Shot 2020-04-19 at 3 00 22 AM" src="https://user-images.githubusercontent.com/935063/79681695-1140a100-81ea-11ea-820a-e33401451af7.png">